config.example.yaml

```diff
@@ -100,7 +100,12 @@ templates:
           - name: action
             type: file
             options:
-              apiURI: http://{domain}/w/api.php
+              apiRequest:
+                method: post
+                uri: '{$.default_uri}'
```

Comment:
For third-party users this is probably puzzling. What is this default_uri and where does it come from? I think it's fine for this patch, but it might be worth revisiting this later.